normandy/base/templatetags/app_data.py

```diff
@@ -0,0 +1,27 @@
+import json
+
+from django.template import Node
+from django.template import Library
+from django.utils.safestring import mark_safe
+
+from normandy.base.api.serializers import UserSerializer
+
+
+register = Library()
+
+
+@register.tag
+def app_data(parser, token):
+    return AppDataNode()
+
+
+class AppDataNode(Node):
```

Comment:
The `takes_context` decorator on that doc page seems like it should do the trick.